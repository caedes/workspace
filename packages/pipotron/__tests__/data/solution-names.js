import solutionNames from '../../src/data/solution-names';

describe('solutionNames', () => {
  it('returns the solutionNames', () => {
    expect(solutionNames).toMatchSnapshot();
  });
});
