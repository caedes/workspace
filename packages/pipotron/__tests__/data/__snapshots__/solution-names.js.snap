// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`solutionNames returns the solutionNames 1`] = `
Array [
  "solutions",
  "issues",
  "problématiques",
  "voies",
  "alternatives",
  "organisations matricielles",
  "améliorations",
  "ouvertures",
  "synergies",
  "actions",
  "options",
  "décisions",
  "modalités",
  "hypothèses",
  "stratégies",
]
`;
